plugins/node/opentelemetry-instrumentation-pg/src/utils.ts

```diff
@@ -156,10 +152,14 @@ export function handleConfigQuery(
     instrumentationConfig.enhancedDatabaseReporting &&
     Array.isArray(queryConfig.values)
   ) {
-    span.setAttribute(
-      AttributeNames.PG_VALUES,
-      arrayStringifyHelper(queryConfig.values)
-    );
+    try {
+      span.setAttribute(
+        AttributeNames.PG_VALUES,
+        JSON.stringify(queryConfig.values)
```

Comment:
values may hold objects or buffers and [PG has some handling](https://node-postgres.com/features/queries#parameterized-query) how this is converted.
Not sure if `JSON.stringify` fits well in that cases.

e.g. `JSON.stringify([Buffer.from("abc")])` results in `'[{"type":"Buffer","data":[97,98,99]}]'`

It differs also from the proposal done in spec repo

Reply:
thanks for this comment.
Do you have any idea what can be used instead of 'stringify'?

Reply:
I think your initial idea to use an array of strings should be fine. Likely some conversion code is needed to handle `null`/`undefined`, buffers, and objects similar as pg does.